manifests/init.pp

```diff
@@ -173,9 +164,6 @@
 # $dhcp::                       Enable DHCP feature
 #                               type:boolean
 #
-# $dhcp_split_config_files::    Split DHCP configuration files. This is needed since version 1.11.
-#                               type:boolean
-#
```

Comment:
maybe also delete $dhcp_vendor some lines below or is that still relevant for 1.11?